pkg/cluster/cluster_config.go

```diff
@@ -34,6 +36,10 @@ var clusterConfig struct {
 	ClusterInfo ClusterInfo
 }
 
+type InstallConfig struct {
+	FIPS *bool `json:"fips"` // Use a pointer to handle cases where fips is not present
```

Comment:
nit: looking at the code, if `fips` is not present, then we default to false so we may not need to use a pointer